Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -440,6 +440,9 @@ open class MapView: UIView {
         displayLink?.invalidate()
         cameraAnimatorsRunner.cancelAnimations()
         cameraAnimatorsRunnerEnablable.isEnabled = false
+
+        let eventsManager = EventsManager.shared(withAccessToken: resourceOptions.accessToken)
+        eventsManager.flush()
```

Comment:
Would it be possible to add a test for this? `EventsManager` is quite "ancient" entity, but it could be retroactively conformed to a protocol(i.e. `EventsManagerProtocol`) and use that instead, which should enable testability.

Reply:
Done